variables.tf

```diff
@@ -42,16 +37,20 @@ variable "autoscale_max" {
 }
 
 variable "autoscale_desired" {
-    default = "4"
+    default = "2"
     description = "Desired autoscale (number of EC2)"
 }
 
-
 variable "instance_type" {
     default = "t2.micro"
 }
 
-variable "ssh_pubkey_file" {
-    description = "Path to an SSH public key"
-    default = "~/.ssh/id_rsa.pub"
+variable "key_name" {
+  description = "Name of AWS pub key pair for intances"
+  default = "some-key-in-aws"
```

Comment:
Re: Will this destroy the key-pair if you call destroy?: No, since the template assumes the key pair is existing and doesn't end up in tfstate.  Were you looking to create one instead?
RE: comment out of date now?: Haven't added a bastion (but I have one in another repo) just locked down ssh by var, which can me modified to only allow your network cidr.  Public bastion with access to private nets, something like sshuttle, would still be a nice add.  Thanks again.